tests/testthat/helper.R

```diff
@@ -43,75 +43,54 @@ map_numbers_to_new_range <- function(numbers, lower, upper) {
 
 }
 
-#' A helper function to get all files in the ~/R/ folder
+#' A helper function that returns the paths of the package files as a vector
 #' @keywords internal
 
-get_R_files <- function() {
-
-  R_files <- base::list.files(path = base::paste0(base::getwd(), "/../../R"),
-                                pattern = "*.R",
-                                full.names = TRUE)
-
-  return(unlist(R_files))
-
-}
-
-#' A helper function to get all files in the ~/tests/testthat/ folder
-#' @keywords internal
-
-get_test_files <- function() {
-
-  test_files <- base::list.files(path = base::getwd(),
-                                 pattern = "*.R",
-                                 full.names = TRUE)
-
-  return(unlist(test_files))
-
-}
-
-#' A helper function to get all files in the ~/man/ folder
-#' @keywords internal
-
-get_man_files <- function() {
+get_visR_files <- function(functions = FALSE,
```

Comment:
For internal functions, I wonder if we should be consistent and use .get_visR_files